templates/block-block-js.mustache

```diff
@@ -0,0 +1,36 @@
+( function( wp ) {
+	var el = wp.element.createElement;
+	var __ = wp.i18n.__;
+
+	// Visit https://wordpress.org/gutenberg/handbook/block-api/ to learn about Block API
+	wp.blocks.registerBlockType( '{{namespace}}/{{slug}}', {
+		title: __( '{{title_ucfirst}}', '{{textdomain}}' ),
```

Comment:
@gziolo This does not seem to be the correct way of defining the textdomain. https://github.com/WordPress/gutenberg/tree/master/i18n

I don't think there is any proper way  of defining the text domain https://github.com/WordPress/gutenberg/issues/4147

Reply:
This was fixed after this PR got merged in https://github.com/wp-cli/scaffold-command/commit/c74032f33bf31f90a8c664ac1498f2456ea98e74